spec/models/claims/training_allowance_spec.rb

```diff
@@ -0,0 +1,168 @@
+require "rails_helper"
+
+RSpec.describe Claims::TrainingAllowance, type: :model do
```

Comment:
I think we might be missing scenarios where there are `internal_draft` or `draft` claims which need to be ignored.

I don't think those status are currently covered in this spec. 👀